docs/source/methods/avas.rst

```diff
@@ -168,6 +168,197 @@ We then use this guess of active orbitals to compute the CASCI energy:::
 For more examples, see :code:`avas-1` to :code:`avas-6` in the :code:`tests/methods` folder.
 In particular, :code:`avas-6` is a practical example on ferrocene.
 
+Defining the molecular plane for π orbitals
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For the above example, the molecule is placed on the yz plane such that the directions of p orbitals
+are aligned with the molecule xyz frame.
+However, sometimes it is hard to make these two xyz frames aligned.
+For example, the molecule contains multiple π subsystems or the plane is not perfect.
+In such cases, if only pi orbitals are desired, we can linearly combine the px, py, and pz orbitals
+to a single 'pz' orbital that perpendicular to the π subsystem plane.
+
+We use the keyword :code:`SUBSPACE_PI_PLANES` to define all the π subsystems planes,
+where each plane is given by a list of atoms' expressions.
+Some valid expressions would be: ::
+
+  - [['C', 'H', 'O']]  # only one plane consisting all C, H, and O atoms of the molecule.
+  - [['C1-6'], ['N1-2', 'C9-11']]  # plane 1 with the first six C atoms of the molecule,
+                                   # plane 2 with C9, C10, C11, N1 and N2 atoms.
+  - [['C1-4'], ['C1-2', 'C5-6']]:  # plane 1 with the first four C atoms of the molecule,
+                                   # plane 2 with C1, C2, C5, and C6 atoms.
+                                   # Two planes share C1 and C2!
+
```

Comment:
@rwashi690  I think you miss the point. Let me rewrite the manual.

This part is just to illustrate the syntax of that keyword.

What I want to achieve is very simple. So suppose using the provided formaldehyde geometry, which is not lying in any of the xy, zx, or yz plane. Now I want AVAS select the pi orbitals of the molecule, which consist of the C and O 2p orbitals that are perpendicular to the plane. Now how to choose these two p orbitals? They are not aligned to x, y, nor z direction. You could try to use ["C(2px)", "O(2px)"] and see what AVAS yields.

There are two ways for this particular case. We could try to rotate the formaldehyde molecule such that the molecular plane is located at xy, xz, or yz plane, and the 2p orbitals are simply aligned with the other direction. For example, if the molecule is at yz plane, the subspace orbitals would be ["C(2px)", "O(2px)"].

The other approach is not to move the molecule but to project all three 2p orbitals onto the plane normal. Suppose the plane normal is (nx, ny, nz). The 2p orbital perpendicular to the plane is simply nx * 2px + ny * 2py + nz * 2pz. This is what I mean px, py, pz orbitals goes to 'pz' orbital that is perpendicular to the plane. Note the quotation mark.

Which approach is more general? I must say it is the second because there might be multiple pi systems in a molecule.